src/audio/module_adapter/iadk/system_agent.cpp

```diff
@@ -16,7 +16,7 @@
 #include <utilities/array.h>
 #include <adsp_error_code.h>
 #include <logger.h>
-#include <system_service.h>
+#include <native_system_service.h>
```

Comment:
Looks like the rimage submodule has been updated and should be another patch unless its needed here ?